src/openrct2-ui/windows/Options.cpp

```diff
@@ -1318,6 +1325,18 @@ static void window_options_mousedown(rct_window* w, rct_widgetindex widgetIndex,
                     window_options_show_dropdown(w, widget, AUTOSAVE_NEVER + 1);
                     dropdown_set_checked(gConfigGeneral.autosave_frequency, true);
                     break;
+                case WIDX_AUTOSAVE_AMOUNT_UP:
+                    gConfigGeneral.autosave_amount += 1;
+                    config_save_default();
+                    gfx_invalidate_screen();
```

Comment:
Sorry for being nitpicky, but it's still not necessary to invalidate the whole screen -- `widget_invalidate()` is enough.
```suggestion
                    widget_invalidate(WIDX_AUTOSAVE);
                    widget_invalidate(WIDX_AUTOSAVE_DROPDOWN);
                    widget_invalidate(WIDX_AUTOSAVE_AMOUNT);
```